apps/etterlatte-behandling/src/main/kotlin/config/ApplicationContext.kt

```diff
@@ -424,6 +430,15 @@ internal class ApplicationContext(
         )
     }
 
+    val saksbehandlerJob: SaksbehandlerJob by lazy {
+        SaksbehandlerJob(
+            saksbehandlerJobService = saksbehandlerJobService,
+            { leaderElectionKlient.isLeader() },
+            0L,
+            interval = if (isProd()) Duration.of(2, ChronoUnit.HOURS) else Duration.of(1, ChronoUnit.HOURS),
```

Comment:
Ville bare satt begge til 1 time.
Hver 2 time er så sjeldent at det er større odds for at vi hadde trigget den gamle løsningen _før_ denne hadde kjørt.. 🤷‍♂️

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/3892/commits/518bd1cc608e1e3b78fd9b0bed3a68ddeb79a640